lib/active_storage/disk_controller.rb

```diff
@@ -4,11 +4,11 @@
 
 require "active_support/core_ext/object/inclusion"
 
-class ActiveStorage::DiskController < ActionController::Base
+class ActiveStorage::DiskController < ActionController::Base # :nodoc:
```

Comment:
I think we should document this class and it's purpose and it's route. Because the rest of the configuration is hidden in the engine definition.